### PR TITLE
Change gencode to handle repairable module; fix build issues

### PR DESCRIPTION
**Pull request by conradoplg**
This changes `gencode` to handle the repairable module: it copies the documentation from `ristretto255` into the other ciphersuite crates.

It's also possible to avoid one other redundancy and copy from `frost-core` into `ristretto255` but that will requires some other tweaks so I'll do that in a separate PR.

I also found some build issues in the ciphersuite-specific repairable modules - we forgot to declare the module so it wasn't being compiled :sweat_smile:

### CODE REVIEW

File: frost-ed25519/src/keys/repairable.rs [hunk 1]
```diff
@@ -1,11 +1,10 @@
-/// Repairable Threshold Schemes
-#![doc = include_str!("../../repairable.md")]
```
Comment (natalieesk) [inline frost-ed25519/src/keys/repairable.rs:3]: Do we not want to generate docs for this? These have been removed in this commit

**Review by natalieesk — CHANGES_REQUESTED** [review]
The build is currently failing on formatting

**Comment by conradoplg** [comment]
> The build is currently failing on formatting

Fixed it, it was an issue with  `gencode` usage (it's run in CI in the `Rustfmt` section, we probably should run it separately...)

File: frost-ed25519/src/keys/repairable.rs (hunk 1 above)
Comment (conradoplg) [reply frost-ed25519/src/keys/repairable.rs:3]: I think that was confusing a comment...

After I saw your comment I copied those docs, so it should look better now

**Comment by dconnolly** [comment]
> > The build is currently failing on formatting
> 
> Fixed it, it was an issue with  `gencode` usage (it's run in CI in the `Rustfmt` section, we probably should run it separately...)

Yeah it's related but not identical, right before might be good (or rustfmt before /and/ after gencode? 🤔)

**Review by dconnolly — APPROVED** [review]

**Pull request merged**